src/main/java/com/iabtcf/ByteBitVector.java

```diff
@@ -28,17 +28,20 @@
     private byte[] buffer;
     private int isrpos;
     private final InputStream is;
+    protected LengthOffsetCache cache;
```

Comment:
Please make this private and accessible through a getter instead of accessing the variable directly.

Reply:
i've changed it to default final. i'm not sure what the concern here is. i am not interested in suggesting that fetching the cache does anything beyond returning the attribute and the usage of this field is limited to a few internal classes.